### DISCUSSION TITLE
The script to install toast componet is incomplete

### DISCUSSION THREAD

**Discussion by andrestejada** [shadcn-ui/ui#566] [General]
When I try to install the toaster with the command 
`npx shadcn-ui add` toast, that creates the toast component, not the toaster, so you need to follow the manual installation to make the toast work.

**Comment by joaopedrodcf** [comment]
Maybe this is not a problem anymore I just test it out and not happening:
<img width="659" alt="image" src="https://github.com/shadcn-ui/ui/assets/11476152/43b7158e-1cac-4c2b-bacc-66fc1ab75dfe">
